torch/tensor.py

```diff
@@ -0,0 +1,365 @@
+import sys
```

Comment:
I didn't look at this, I assume it's just the same as what was in variable.py, but let me know if that's not the case.

Reply:
It's the same aside for mechanical conversions from Variable -> Tensor and deleted stale documentation.